### ISSUE TITLE
Missing old patches in the changelog popup

### ISSUE THREAD

**Issue by speed2CZ** [FAForever/fa#2647]
Looking for a volunteer to feed this file: https://github.com/FAForever/fa/blob/deploy/fafdevelop/lua/ui/lobby/changelogData.lua
with older patches.

It's a labour work so that's why I won't do it :P
The order of the tables is descending, from the newest patch to the oldest one.

Example:
```LUA
    {
        version = 3701,
        name = 'Game Version 3701',
        description = {
            '- Fixed bug that caused the Kennel structure to stay after being upgraded which resulted into having two buildings stacked upon each other',
            '- Changed Novax Sat fix in order to not potentially break mods',
            '- Updated mod blacklist',
            '- Removed smoke effect on the water surface caused by wrecks on the bottom of the ocean',
            '- Moved current patch number in the lobby to a more visible location',
            '- Allow Users to open the Changelog by clicking on the patch number in the game',
            '- Allow Users to open the changelog on github by adding a button to the changelog popup in the lobby',
            '',
            '**Contributors**',
            '- speed2',
            '- Strogo',
            '- Exotic-Retard',
            '- PhilipJFry',
        },
},
```
You can get the older changelog here: https://github.com/FAForever/fa/blob/develop/changelog.md

Have fun whoever picks it :)

**Comment by OCGDFS** [comment]
made a script. should work for future changelogs

[convertReadme.zip](https://github.com/FAForever/fa/files/2728477/convertReadme.zip)

**Comment by Garanas** [comment]
Closed as #3369 is merged.
